---
layout: archive
title: "About me"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

I am a fourth-year Ph.D. student, advised by Prof. [Fengzhu Sun](https://dornsife.usc.edu/labs/fsun), from [Department of Quantitative 
and Computational Biology](https://www.qcb-dornsife.usc.edu/) at University of Southern California. 
My research focuses on the bioinformatics and biostatistics.
I spent my entire childhood in Shanghai, China until my family moved to Wuhan, China when I stepped into the middle school. 
I came to America in 2019 and am now living in Los Angeles, USA.

# Education
* B.S. in Mathematics and Statistics, Wuhan University, 2019
* Secondary Degree in Computer Science, Wuhan University, 2019
* Ph.D in Bioinformatics, University of Southern California, 2024 (expected)

# Reseach interest
* Metagenomics and metagenomic Hi-C
* Single-cell omics
* High dimensional statistics 
* Network analysis

# Publication
[1] **Yuxuan Du**, Sarah M. Laperriere, Jed Fuhrman, and Fengzhu Sun. HiCzin: Normalizing metagenomic Hi-C data and detecting spurious contacts using zero-inflated negative binomial regression. RECOMB 2021. [[Paper](https://www.biorxiv.org/content/10.1101/2021.03.01.433489v1.abstract)] [[Code](https://github.com/dyxstat/HiCzin)] [[Conference](https://www.recomb2021.org/accepted-papers)] [[Video Record](https://www.youtube.com/watch?v=VjFIYcrtPi8)] 

[2] **Yuxuan Du**, Sarah M. Laperriere, Jed Fuhrman, and Fengzhu Sun. Normalizing Metagenomic Hi-C Data and Detecting Spurious Contacts Using Zero-Inflated Negative Binomial Regression. Journal of Computational Biology, 2022. [[Paper](https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0439)] [[Code](https://github.com/dyxstat/HiCzin)] 

[3] **Yuxuan Du** and Fengzhu Sun. HiCBin: binning metagenomic contigs and recovering metagenome-assembled genomes using Hi-C contact maps.
Genome Biology, 2022. 
[[Paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02626-w)] [[Code](https://github.com/dyxstat/HiCBin)]

[4] **Yuxuan Du** and Fengzhu Sun. HiFine: integrating Hi-c-based and shotgun-based methods to reFine binning of metagenomic contigs. 
Bioinformatics, 2022. [[Paper](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac295/6575440?login=true)] [[Code](https://github.com/dyxstat/HiFine)]

[5] **Yuxuan Du**, Jed A. Fuhrman, and Fengzhu Sun. ViralCC retrieves complete viral genomes and virus-host pairs from metagenomic Hi-C data. Nature Communications, 2023. [[Paper](https://www.nature.com/articles/s41467-023-35945-y)] [[Code](https://github.com/dyxstat/ViralCC)]

[6] **Yuxuan Du**, Jieli Ding, Jiaqian Zhang, Yanqin Feng. Quadratic upper bound algorithms for estimation under cox model in case-cohort studies.
Statistics and Its Interface, 2022. [[Paper](https://www.intlpress.com/site/pub/pages/journals/items/sii/_home/acceptedpapers/index.php)]

# In Submission
[7]  **Yuxuan Du** and Fengzhu Sun. MetaCC: an efficient and integrative framework for analyzing both long-read and short-read metagenomic Hi-C datasets.
In submission.


# Professional Expericence
**Oral presentation**: HiCzin Normalizing Metagenomic Hi-C Data and Detecting Spurious Contacts Using Zero-Inflated Negative Binomial Regression.
[RECOMB 2021](https://www.recomb2021.org/), Augest 31st, Padova, Italy. [[Video Record](https://www.youtube.com/watch?v=VjFIYcrtPi8)]

**Reviewer**: [RECOMB 2021](https://www.recomb2021.org/), [RECOMB 2022](https://recomb2022.net/), [RECOMB 2023](https://recomb2023.net/), [Frontiers in Cellular and Infection Microbiology](https://www.frontiersin.org/journals/cellular-and-infection-microbiology).

# Contact and Service
If you have any collaborations, service requests, or questions, welcome to contact me by email **yuxuandu@usc.edu**. 
 
